server/src/main/java/org/apache/uniffle/server/storage/SingleStorageManager.java

```diff
@@ -113,6 +115,22 @@ public void updateWriteMetrics(ShuffleDataFlushEvent event, long writeTime) {
     }
   }
 
+
+  @Override
+  public boolean canWrite(ShuffleDataFlushEvent event) {
+    try {
+      Storage storage = selectStorage(event);
+      // if storage is null, appId may not be registered
+      if (storage == null || !storage.canWrite()) {
+        return false;
+      }
+      return true;
+    } catch (Exception e) {
+      LOG.warn("", e);
```

Comment:
?

Reply:
What do you mean? RuntimeException will be throw if  storage is corrupted in ` LocalStorageManager`,  I think we should catch it here, for other  StorageManager maybe can be writed.  Do you mean we should use LOG.error?

Reply:
We shouldn't log empty message.